File: C/Lv.0/8.c
//두 수의 나눗셈
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int num1, int num2) {
    int answer = 0;
    
    answer = (double)num1 / num2 * 1000;
    answer = (int)answer;
    
    return answer;
}

File: Python/Lv.0/8.py
//OX퀴즈
def solution(quiz):
    answer = []
    
    for i in range(len(quiz)):
        for j in range(len(quiz[i])):
            if(quiz[i][j] == '='):
                if(eval(quiz[i][:j]) == eval(quiz[i][(j + 1):])):
                    answer.append("O");
                else:
                    answer.append("X");
                break;
    
    return answer